### PR TITLE
[webview_flutter_android] Change relative path to uri in generated pigeon test file

### PR DESCRIPTION
**Pull request by bparrishMines**
This fixes the issue of relative paths in a generated test file that prevent `mockito` code generation. Also bumps `mockito` version to `5.1.0`.

No version change: Only affects plugin development

Cause: https://github.com/flutter/flutter/issues/97744 that causes https://github.com/flutter/flutter/issues/99766
Fixes https://github.com/flutter/flutter/issues/99766

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Review by mvanbeusekom — APPROVED** [review]
LGTM

File: packages/webview_flutter/webview_flutter_android/README.md [hunk 1]
```diff
@@ -11,6 +11,21 @@ normally. This package will be automatically included in your app when you do.
 
 This package uses [pigeon][3] to generate the communication layer between Flutter and the host platform (Android). The communication interface is defined in the `pigeons/android_webview.dart` file. After editing the communication interface regenerate the communication layer by running the `./generatePigeons.sh` shell script.
 
+Due to [flutter/flutter#97744](https://github.com/flutter/flutter/issues/97744), the generated test
+pigeon file needs to update one of its imports to properly work with `mockito`.
```
Comment (stuartmorgan) [inline packages/webview_flutter/webview_flutter_android/README.md:15]: s/needs to update of its imports/needs one of its imports updated/ (the file isn't going to do the update :) )

File: packages/webview_flutter/webview_flutter_android/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 2.8.4
```
Comment (stuartmorgan) [inline packages/webview_flutter/webview_flutter_android/CHANGELOG.md:1]: You can make this a NEXT; it only affect developers of the plugin.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/webview_flutter/webview_flutter_android/pubspec.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ name: webview_flutter_android
 description: A Flutter plugin that provides a WebView widget on Android.
 repository: https://github.com/flutter/plugins/tree/main/packages/webview_flutter/webview_flutter_android
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+webview%22
-version: 2.8.3
+version: 2.8.4
```
Comment (stuartmorgan) [inline packages/webview_flutter/webview_flutter_android/pubspec.yaml:5]: Revert to 2.8.3

**Review by stuartmorgan — APPROVED** [review]
LGTM with version bump removed.

**Pull request merged**